changes/1927-email-template-images

```diff
@@ -0,0 +1 @@
+Move email template images from github to fleetdm.com; update links
```

Comment:
Nitpick: I think we've used `*` rather than `+` for the list. The reason I mention it is because we use `make changelog` to collect these things, and the more uniform we write them, the easier it'll be to create the changelog afterwards.